kusto/query_builder.go

```diff
@@ -506,6 +507,143 @@ func NewStmt(query stringConstant, options ...StmtOption) Stmt {
 	return s
 }
 
+func (s Stmt) AddDatabase(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddTable(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddColumn(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) AddFunction(query string) Stmt {
+	return s.SafeAddName(query)
+}
+
+func (s Stmt) SafeAddName(query string) Stmt {
+	if !(strings.Contains(query, "'")) {
```

Comment:
You reordered the logic in a way which I can't say is correct.
Also, please do support this C# code section 

```
if (!forceNormalization
                && IsIdentifier(name)
```
The ask to exclude was only for 
```
&& !IsKeyword(name) 
```